about.html

```diff
@@ -20,6 +20,15 @@
     <a href="mailto:{{site.email}}" class="contact__item contact__item--email">
       <h4>Email</h4>
     </a>
+
+  </div>
+
+  <h2> Stickers! </h2>
```

Comment:
Could you please replace "Stickers" by "Merchandise" and add an additional item for our [Spreadshirt shop](https://shop.spreadshirt.co.uk/typelevel/)?